### DISCUSSION TITLE
Is this the roadmap area? Idea: User Color Libraries

### DISCUSSION THREAD

**Discussion by billiemead** [givanz/VvvebJs#348] [Ideas]
What do you think of a way to save color libraries... maybe as a .json file that can be re-used for projects?

**Comment by givanz** [comment]
I added color palette support in a recent commit https://github.com/givanz/VvvebJs/commit/0e2ea3ef40523cc519fde6be5262ffb5149eceab

On the roadmap there is a Theme Style feature that will let user choose color palettes and font styles defined by the theme.

A user defined color library where users can save favorites colors to show in the color input palette dialog is a good idea and will be implemented after theme styles is added. 

Thanks for the suggestion.
